tests/functional/run.sh

```diff
@@ -7,7 +7,9 @@ TESTS="TestManyBricksVolume $TESTS"
 
 # install glide
 if ! command -v glide ; then
-    curl https://glide.sh/get | sh
+	echo glide is not installed
+	echo 'run "curl https://glide.sh/get | sh" if you are crazy'
```

Comment:
please use more appropriate language ;-)

In the main Makefile we already have:

> Please install glide.
> Install it using your package manager or
> by running: curl https://glide.sh/get | sh.

this may be not perfect but I think it's better.